src/app/shared/entry.ts

```diff
@@ -220,7 +220,15 @@ export abstract class Entry implements OnInit, OnDestroy {
       return null;
     }
     const selectedTag = this.selectVersion(versions, urlVersion, defaultVersion);
-    return selectedTag || versions.reduce((a, b) => (b.last_built > a.last_built ? b : a));
+    return (
+      selectedTag ||
+      versions.reduce((a, b) => {
+        if (!b.last_built && !a.last_built) {
```

Comment:
A comment might be nice, e.g., newly created versions don't have a last_built (if that's what's going on).